XCProject.cs

```diff
@@ -378,6 +393,22 @@ public PBXDictionary AddFile( string filePath, PBXGroup parent = null, string tr
 							string frameworkPath = Path.Combine( "$(SRCROOT)", Path.GetDirectoryName( filePath ) );
 							this.AddFrameworkSearchPaths(new PBXList(frameworkPath));
 						}
+						
+						if(embed)
+						{
+							string ldRuntimePath = "$(inherited) @executable_path/Frameworks";
+							this.AddLDRuntimeSearchPaths(ldRuntimePath);
+							//LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";

```

Comment:
Yep.  It is also possible we could remove the entire AddLDRuntimeSearchPaths method since it should work without it.